doc/conf.py

```diff
@@ -326,5 +326,10 @@ def setup(app):
     'examples_dirs': examples_dirs,
     'gallery_dirs': gallery_dirs,
     'find_mayavi_figures': find_mayavi_figures,
-    'expected_failing_examples': ['../examples/plot_raise.py', '../examples/plot_syntaxerror.py']
+    'expected_failing_examples': ['../examples/plot_raise.py', '../examples/plot_syntaxerror.py'],
+    'binder_org': 'choldgraf',
```

Comment:
I'll have to work on this myself. But to show this feature we'll need to move our documentation out of readthedocs into github-pages. I've had that in mind also to show our support for mayavi. This PR is a great reason to do that.